browser-test/src/support/admin_predicates.ts

```diff
@@ -178,8 +229,58 @@ export class AdminPredicates {
     const valuesToSet = this.coalesceValueOptions(complexValues, values, value)
     let groupNum = 1
     for (const valueToSet of valuesToSet) {
-      await this.fillValue(scalar, valueToSet, groupNum++, questionId)
+      await this.legacyFillValue(scalar, valueToSet, groupNum++, questionId)
+    }
+  }
+
+  /**
+   * Configures a(n expanded form logic) predicate with the given inputs. For the values, populate whatever is present.
+   */
+  async configureSubcondition({
+    conditionId,
+    subconditionId,
+    questionText,
+    scalar,
+    operator,
+    value,
+  }: SubconditionSpec) {
+    await this.selectQuestion(conditionId, subconditionId, questionText)
+
+    // In case we want to populate default scalar and operator values.
+    if (scalar) {
+      await this.selectScalar(conditionId, subconditionId, scalar)
+    }
+
+    if (operator) {
+      await this.selectOperator(conditionId, subconditionId, operator)
+    }
+
+    await this.fillValue(conditionId, subconditionId, value)
+  }
+
+  /**
+   * Asserts the state of a given subcondition, checking selected question, scalar, operator, and value(s).
+   */
+  async expectSubconditionEquals({
+    conditionId,
+    subconditionId,
+    questionText,
+    scalar,
+    operator,
+    value,
+  }: SubconditionSpec) {
+    await this.expectSelectedQuestion(conditionId, subconditionId, questionText)
+
+    // Operator and scalar might just be default values.
+    if (scalar) {
+      await this.expectSelectedScalar(conditionId, subconditionId, scalar)
     }
+
+    if (operator) {
+      await this.expectSelectedOperator(conditionId, subconditionId, operator)
+    }
+
+    await this.expectFilledValue(conditionId, subconditionId, value)
   }
```

Comment:
This block is a good place to start reviewing - most of the new helper methods just get called through these two.